libvast/src/taxonomies.cpp

```diff
@@ -35,18 +36,20 @@ caf::error convert(const data& d, concepts_type& out) {
     auto name = caf::get_if<std::string>(&n->second);
     if (!name)
       return make_error(ec::convert_error, "name is not a string:", *n);
+    auto& dest = out.data[*name];
     auto fs = c->find("fields");
-    if (fs == c->end())
-      return make_error(ec::convert_error, "concept has no fields:", d);
-    if (const auto& fields = caf::get_if<list>(&fs->second)) {
-      for (auto& f : *fields) {
-        auto field = caf::get_if<std::string>(&f);
-        if (!field)
-          return make_error(ec::convert_error, "field is not a string:", f);
-        out.data[*name].fields.push_back(*field);
+    if (fs != c->end()) {
+      if (const auto& fields = caf::get_if<list>(&fs->second)) {
+        for (auto& f : *fields) {
+          auto field = caf::get_if<std::string>(&f);
+          if (!field)
+            return make_error(ec::convert_error, "field is not a string:", f);
```

Comment:
This error message is not user-facing: The user does not know, what `field` is here.